preview-env/create/action.yml

```diff
@@ -69,9 +69,40 @@ runs:
   - name: Deploy
     shell: bash
     run: >-
-      argocd app create ${{ inputs.argocd_arguments }}
+      if argocd app create ${{ inputs.argocd_arguments }}; then
```

Comment:
not really a fan of it being as part of a if statement, maybe run it and follow up with `$?`